test/unit/sql/add-constraint.test.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+/* jshint -W030, -W110 */
+const Support   = require(__dirname + '/../support');
+const current   = Support.sequelize;
+const expectsql = Support.expectsql;
+const sql = current.dialect.QueryGenerator;
+const expect = require('chai').expect;
+const dialect = Support.getTestDialect();
+
+if (dialect !== 'sqlite') {
```

Comment:
sqlite implementation of `addContraint` is a bit different. It has its own queryInterface method and queryGenerator functions. So, i've skipped them. But i'll go ahead and add unit tests for sqlite queryGenerator functions.